src/components/BounceRulesContainer/index.jsx

```diff
@@ -194,16 +155,28 @@ const BounceRulesContainer = ({
               </div>
             )}
           {isRulesEmpty &&
-            !isFetching && (
+            !isFetching &&
+            isBounceRulesTab && (
               <div {...WriteSelectors.emptyRulesWarning}>
                 <EmptyRules />
               </div>
             )}
+          {isActivityLogTab &&
+            !isActivityEmpty && (
+              <div {...WriteSelectors.activityTable}>
+                <ActivityLogContainer
+                  activityLog={filteredActivityLog}
+                  handleActivityClicked={handleActivityClicked}
+                />
+              </div>
+            )}
         </Column>
       </Row>
       <Row>
         <Column width={4} offset={5}>
-          {!isRulesEmpty &&
+          {isBounceRulesTab &&
```

Comment:
would there ever be a chance both `isBounceRulesTab` and `isActivityLogTab` are both true? if not, we can remove the `! isActivityLogTab` as part of this conditional check since it would be redundant of `isBounceRulesTab`